### DISCUSSION TITLE
How to set up terragrunt to use the new testing framework released with terraform 1.6

### DISCUSSION THREAD

**Discussion by mrzonca** [gruntwork-io/knowledge-base#775] [How do I]
Hello, 
With the release of Terraform 1.6, Hashicorp enabled testing of configurations using new framework. I would like to use this solution in my Terragrunt configuration. Is there documentation on the Gruntwork side showing examples of using the new framework instead of Terratest? Or any related external documentation. 

Best, 
Mateusz

---

<ins datetime="2023-10-09T11:20:16Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110515">Tracked in ticket #110515</a></p>
</ins>

**Comment by denis256** [comment]
Hello,
As of now, support for Terraform 1.6 and the testing framework has not been implemented

So far we have only Official documentation:
https://developer.hashicorp.com/terraform/cli/test
https://developer.hashicorp.com/terraform/tutorials/configuration-language/test

References:
https://github.com/gruntwork-io/terragrunt/issues/2747
https://github.com/gruntwork-io/terratest/issues/1357
